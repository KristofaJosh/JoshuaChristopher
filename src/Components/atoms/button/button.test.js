import React from "react";
import renderer from "react-test-renderer";
import { ThemeProvider } from "styled-components";
import Button from "./index";


it("renders correctly when called ", () => {
    const tree = renderer
        .create(
            <ThemeProvider theme={{mode: 'light'}}>
                <Button/>
            </ThemeProvider>
        )
        .toJSON();
    expect(tree).toMatchSnapshot();
});