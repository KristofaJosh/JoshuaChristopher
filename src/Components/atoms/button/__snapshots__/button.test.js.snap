// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when called  1`] = `
<button
  className="sc-AxirZ eGsmtj"
>
  <p
    className="sc-AxjAm jA-DHev"
  >
    <span>
       /&gt;
    </span>
  </p>
</button>
`;
